src/git_provider/mod.rs

```diff
@@ -47,11 +48,27 @@ pub trait GitProvider {
     fn set_token(&self, path: &PathBuf) -> Result<String>;
     fn get_token(&self) -> Result<String>;
 
+    fn set_title(&self, branch: &String) -> Result<String> {
+        let title = dialoguer::Input::<String>::with_theme(&ColorfulTheme::default())
+            .with_prompt("Pull request title:")
+            .default(branch.to_owned())
+            .interact()?;
+
+        Ok(title)
+    }
```

Comment:
I would like that the PR title would coinicide with the Commit title, not with the branch name. You can use `git_client.get_current_commit_title`.